pam/vplugin/vplugin.go

```diff
@@ -112,6 +112,10 @@ func (vp *VPluginRPC) GetGameRound(req pam.GetGameRoundRequest) *pam.GameRoundRe
 	return &response
 }
 
+func (vp *VPluginRPC) GetSettlementType() string {
+	return "mixed" // TODO: Hard coded for now, plugin server init could return settlement type from configuration.
```

Comment:
Noticed it has "mixed" here, is that intentional? Asking since `gamewise` is default

Reply:
Maybe I got confused.. So if pam settlement type = mixed, then we would attempt to include round transactions. Isn't it the other way around though, if the pam does gamewise settlement it needs additional data on settlement? 😆

Reply:
Great that you spotted this, I am the source of the confusion since I mixed "mixed" and "gamewise" up totally. 🤣 I've basically inverted everything now so default should be mixed (i.e old behavior) and gamewise should include round transactions. It is hard coded to gamewise in the vplugin.